standard/expressions.md

```diff
@@ -4793,7 +4792,7 @@ It is possible for anonymous function delegates to share some captured variables
 
 > *Example*: For example, if `F` is changed to
 >
-> <!-- Example: {template:"standalone-console", name:"InstantiationOfLocalVariables6", IgnoredWarnings:["CS8321"], additionalFiles:["DvoidNoArgs.cs"]} -->
+> <!-- Example: {template:"code-in-partial-class", name:"InstantiationOfLocalVariables6", IgnoredWarnings:["CS8321"], additionalFiles:["Caller.cs"], inferOutput: true} -->
```

Comment:
I think you've forgotten to add Caller.cs...

Reply:
@jskeet OK, done. Now we no longer need additional-file DvoidNoArgs.cs, but I don't see how to remove that.

Reply:
> Now we no longer need additional-file DvoidNoArgs.cs, but I don't see how to remove that.

There's a "delete file" option in GitHub in the three dots on the top right corner. I can do it if you can't find the option easily.